src/api/java/mezz/jei/api/gui/handlers/IGuiContainerHandler.java

```diff
@@ -45,8 +45,23 @@ default Object getIngredientUnderMouse(T containerScreen, double mouseX, double
 	/**
 	 * Return the JEI-controlled clickable areas for this GUI.
 	 * This is useful when you want to add a spot on your GUI that opens JEI and shows recipes.
+	 *
+	 * @deprecated Prefer and implement {@link #getGuiClickableArea(ContainerScreen, double, double)} instead.
 	 */
+	@Deprecated//TODO 1.16: Remove this in favor of getGuiClickableArea, changing that to default to returning null
```

Comment:
No need for a TODO here, anything deprecated gets audited and should be removed when it seems like a good time to (usually the next version). Sometimes things are left deprecated longer to help people porting.